ports/atmel-samd/mpconfigport.h

```diff
@@ -112,6 +112,11 @@
 #define CIRCUITPY_DEFAULT_STACK_SIZE                4096
 #endif
 
+#ifndef BOARD_BROWNOUT_LEVEL (
+#define BOARD_BROWNLOUT_LEVEL (39)
```

Comment:
Since the `SAMD21` and `SAMD5xE5x` levels are different units, I think it would be better to have `SAMD21_BOD33_LEVEL` and `SAMD5x_E5x_BOD33_LEVEL` definitions, to avoid confusion. Note I also label it as BOD33, to be clearer.